src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1536,7 +1536,12 @@ The file `PROACTIVE_HOME/config/security.java.policy-server` allows to configure
 
 * Deploy ProActive Nodes
 * Execute jobs
-* Pause the Scheduler
+* Perform administrator actions from the scheduler portal like:
+** Start the scheduler: Jobs can be submitted. Getting the jobs results is possible.
+** Stop the scheduler: Jobs cannot be submitted anymore. This terminates every submitted jobs. Getting the jobs results is possible.
+** Freeze the scheduler: Every running tasks are terminated, but the pending jobs wait for the scheduler to resume.
+** Pause the scheduler: Every running jobs are terminated.
```

Comment:
I think there could be a misunderstanding with the word 'terminated'. It could be interpreted like 'killed'. Maybe saying something like 'run to completion' would be clearer. What do you think?

Reply:
ok

Reply:
You could say that it's similar to stop, but that jobs can still be submitted (and will remain in pending state until the scheduler is resumed).